sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -8,18 +8,22 @@ EAPI=6
 EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
+PYTHON_COMPAT=( python2_7 )
```

Comment:
No. `python_setup` is called only conditionally. Furthermore, there are proper deps in `DEPEND`.